src/Main.java

```diff
@@ -1,114 +1,64 @@
-import java.util.Arrays;
-import java.util.Random;
-import java.util.Scanner;
+import java.util.*;
 
 public class Main {
     public static void main(String[] args) {
-        String hello = "Hello World!";
-        ans(2);
-        findSymbolOccurance(hello, 'h');
-        ans(3);
-        System.out.println(findWordPosition("Apple", "Plant"));
-        ans(4);
-        System.out.println(stringReverse(hello));
-        ans(5);
-        System.out.println(isPalindrome("ERE"));
-        System.out.println(isPalindrome("Allo "));
-        System.out.println();
-        System.out.println("THE GAME");
-        game();
+        String[] words = new String[14];
+        Arrays.fill(words, "apple");
+        System.out.println("Occurance " + countOccurance(words, "apple"));
+        System.out.println(toList(words));
+        Integer[] integers = {3, 4, 5, 2, 4, 66, 3, 23, 6, 4};
+        System.out.println("Only unique ints: " + findUnique(integers));
+        List<String> words2 = List.of("apple", "bird", "fox", "fox", "apple", "apple", "bird", "bear", "bear", "cat", "frog", "frog", "frog", "frog", "frog");
+        calcOccurance(words2);
+        System.out.println(findOccurance(words2));
     }
 
 
-    public static void findSymbolOccurance(String str, char letter) {
+    public static int countOccurance(String[] words, String target) {
         int occurance = 0;
-
-        char[] chars = str.toLowerCase().toCharArray();
-        for (char character : chars) {
-            if (letter == character) {
-                occurance += 1;
+        for (String word : words) {
+            if (word.equals(target)) {
+                occurance++;
             }
         }
-        System.out.println("Occurrence \"" + letter + "\" in " + str + " = " + occurance);
-
-    }
-
-    public static int findWordPosition(String source, String target) {
-        int pos;
-
-        pos = source.toLowerCase().indexOf(target.toLowerCase());
-        return pos;
+        return occurance;
     }
 
-    public static String stringReverse(String str) {
-        StringBuilder reversed = new StringBuilder(str);
-        reversed.reverse();
-        return reversed.toString();
+    public static List<Object> toList(Object[] toConvert) {
```

Comment:
Object is not a particular type
```
    String[] arr = {"a", "b"};
    List<String> afterToList = toList(arr); - it will not work
```
to make it work consider using generics